README.md

```diff
@@ -1,2 +1,36 @@
 # hw01
 Repository for homework 1
+
+Text added through RStudio
+
+A brief biography as a part of the homework:
+
+## Zahra Amalia Syarifah
+I was born in Indonesia and grew up there. However, shortly before my high school graduation I moved to the UK. *So I **never** actually completed high school*. I went straight to a college in Southeast England and moved to Scotland for my undergraduate. Now, I am completing my Master in the States.
+
+For the sake of using the list. Here are the list of my favourite carbohydrates:
+* Pasta
+* Potato
+* Bread
+
+Here is an image of Hungarian Goulash (for the sake of the points, but also because it's one of my favourite dishes):
+
+![Image of Hungarian Goulash](http://img.taste.com.au/JCSA2ji6/w720-h480-cfill-q80/taste/2016/11/goulash-soup-75934-1.jpeg)
+
+You might want to make it at home, so here is a recipe for it:
+https://www.daringgourmet.com/traditional-hungarian-goulash-gulyas/ 
+[Recipe](https://www.daringgourmet.com/traditional-hungarian-goulash-gulyas/)
+
+# Reporting Process
+1. Using the forked and cloned version, I loaded README.md from the 'Files'.
+2. After I made the changes I saved it.
+3. README.md appeared under the 'Git' tab.
+4. Then I clicked 'Pull'.
+5. Then I selected 'README.md' and clicked 'Commit'.
+6. After making notes and checking the changes, I clicked 'Commit' and 'Push'.
+
+## Reflections
```

Comment:
Zahra, I had a similar experience with this homework! A lot of what I was trying to do felt very complicated, especially the process of commits and pushes. But once I figured it out, it seemed so simple. The pain and blood was definitely worth it, and, I too, hope it gets easier.